README.md

```diff
@@ -16,11 +16,57 @@ To install, simply add it to your `composer.json` file:
 $ composer require myonlinestore/omnipay-klarna-checkout
 ```
 
+## Initialization
+
+```php
+$gateway = Omnipay::create('\MyOnlineStore\Omnipay\KlarnaCheckout\Gateway');
+
+$gateway->initialize(['username' => $username, 'secret' => $secret]);
```

Comment:
The following optional parameters are also available:

```suggestion
$gateway->initialize([
   'username' => $username,
   'secret' => $secret,
   'api_region' => $region, // May be Gateway::API_VERSION_EUROPE (default) or Gateway::API_VERSION_NORTH_AMERICA
   'testMode' => false // default: true
]);
```

Reply:
Awesome!